.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Related Issue
+- Issue goes here with URL/Number
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Checklist
+- [ ] Follow [Google developer documentation style guide](https://developers.google.com/style)
+- [ ] Issues must be Listed
+
+# Screenshots
```

Comment:
I think it would be best if we delete the screenshot section for the following reasons:

1. I don't think expecting a screenshot is accessible for all users (for example screenreader users)
2. It's unnecessary as we have a deploy preview

Reply:
Sure, That's good reason to delete it.